docs/proposals/kubelet-qos.md

```diff
@@ -0,0 +1,142 @@
+# Quality of Service in Kubernetes

```

Comment:
Can I suggest as an alternative, "Pod Resource Quality of Service"?  The term "resource" is also overloaded (I read it and thought 'API resource').